Web.md

```diff
@@ -124,6 +124,81 @@ MIME type mismatch or `opaque` response types
 [reject](http://tc39.github.io/ecma262/#sec-rejectpromise) the Promise with a
 `WebAssembly.CompileError`.
 
+### Structured Clone of a `WebAssembly.Memory`
```

Comment:
This should be Structured serialization; structured cloning is no longer a thing at the spec level (although it can be used as an informal term outside of specs).